framework/src/postprocessors/SideAverageValue.C

```diff
@@ -19,6 +19,9 @@ InputParameters
 validParams<SideAverageValue>()
 {
   InputParameters params = validParams<SideIntegralVariablePostprocessor>();
+  params.addClassDescription("Computes the integral of an variable on a sideset using an area or "
```

Comment:
Computes the average value of a variable on a sideset. Note that this cannot be used on the centerline of an axisymmetric model.